src/sinks/datadog/metrics/encoder.rs

```diff
@@ -795,9 +811,12 @@ mod tests {
         let result = encoder.finish();
         assert!(result.is_ok());
 
-        let (_payload, mut processed) = result.unwrap();
+        let (payload, mut processed, raw_bytes) = result.unwrap();
         assert_eq!(processed.len(), 1);
         assert_eq!(expected, processed.pop().unwrap());
+
+        assert_eq!(81, payload.len());
+        assert_eq!(70, raw_bytes);
```

Comment:
I'm wondering if this is correct and if there is a way to verify it?  The raw_bytes are smaller than the compressed payload.

Is this just a case of protobuf not compressing well?

Reply:
That's definitely a little surprising, but sketches are indeed mostly integer data and aren't typically very compressible. This could definitely be a case of the compression dictionary, etc, ending up adding overhead.... especially given how small the input payload is.